exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -173,21 +202,25 @@ func (prwe *prwExporter) PushMetrics(ctx context.Context, md pmetric.Metrics) er
 		return errors.New("shutdown has been called")
 	default:
 
-		tsMap, err := prometheusremotewrite.FromMetrics(md, prwe.exporterSettings)
-		if err != nil {
-			prwe.telemetry.recordTranslationFailure(ctx)
-			prwe.settings.Logger.Debug("failed to translate metrics, exporting remaining metrics", zap.Error(err), zap.Int("translated", len(tsMap)))
+		// If feature flag not enabled support only RW1
+		// TODO remove when feature flag removed
```

Comment:
TODO isn't necessary. At least to me, this is obvious.
```suggestion
		// If feature flag not enabled support only RW1
```